xAPI.md

```diff
@@ -2483,8 +2483,18 @@ See <a href="#AppendixE">Appendix E: Example Signed Statement</a> for an example
 
 ## 3.0 Metadata
 
+Metadata is additional information about the resource.  It enables decision making, search, and discoverability within 
+systems.  In xAPI, metadata can be utilized in a variety of locations.  The most common is within <a name="actdef"> Activity Definitions </a>, 
+where the metadata describes the Activity. Most types of metadata have the opportunity to have Extensions.  
```

Comment:
I'd delete "Most types of metadata have the opportunity to have Extensions." as it's added complexity that's not needed here and covered in detail elsewhere.